Added ALT Tags for install/desktop-msi-installer-and-group-policy-ins… (#5597)
mattermost/docs: source/install/desktop-msi-installer-and-group-policy-install.rst

1. Using a newly created Windows VM or dedicated Windows computer, make sure to use a Windows version that supports ``Edit group policy`` out of the box (i.e. Windows 10 Pro or Enterprise).

   .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00001.png

>       :alt: Make sure to use a Windows version that supports Edit Group Policy

Reviewer: ```suggestion
      :alt: Make sure to use a Windows version that supports ``Edit group policy``.
```

2. Navigate to the `Mattermost Desktop <https://github.com/mattermost/desktop>`__ repo on `Github.com <https://github.com/>`__.

   .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00002.png

>       :alt: Navigate to the `Mattermost Desktop repo on `Github.com

Reviewer: ```suggestion
      :alt: Navigate to the Mattermost Desktop repository on GitHub.
```

3. Navigate to the release page for `version v5.0.4 <https://github.com/mattermost/desktop/releases/tag/v5.0.4>`__ and download the appropriate installer for your version of Windows (32bit vs. 64bit).

4. Download the `source.zip <https://github.com/mattermost/desktop/archive/v5.0.4.zip>`__ file as well to extract group policy files.

   .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00003.png

>       :alt: Download the source.zip file as well to extract group policy files.

Reviewer: ```suggestion
      :alt: Navigate to the release page for the latest release and download the installer for your version of Windows. Download the source.zip file as well to extract group policy files.
```

Installing group policy files locally
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

1. Browse to the folder the above files were downloaded to and unzip the ``desktop-5.0.4.zip`` file in place.

   .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00004.png

>       :alt: Browse to the folder the above files were downloaded to and unzip the desktop-5.0.4.zip file in place

Reviewer: ```suggestion
      :alt: Browse to the download folder and unzip the ZIP file.
```

2. Navigate to the unzipped ``desktop-5.0.4\resources\windows\gpo`` folder and copy the contents.

   .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00005.png 

>       :alt: Navigate to the unzipped desktop-5.0.4\resources\windows\gpo folder and copy the contents.

Reviewer: ```suggestion
      :alt: Navigate to \resources\windows\gpo directory and copy its contents.
```

3. Navigate to the ``C:\Windows\PolicyDefinitions`` folder and paste the files copied in the last step. 

   .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00006.png

>       :alt: Navigate to the C:\Windows\PolicyDefinitions folder and paste the files copied in the last step.

Reviewer: ```suggestion
      :alt: Navigate to the Windows\PolicyDefinitions directory and paste the files you copied as part of the last step.
```

4. Verify the ``mattermost.admx`` file is in the ``C:\Windows\PolicyDefinitions`` folder.

   .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00007.png

>       :alt: Verify the mattermost.admx file is in the C:\Windows\PolicyDefinitions folder.

Reviewer: ```suggestion
      :alt: Verify the mattermost.admx file is present in the Windows\PolicyDefinitions directory.
```

5. Verify the ``mattermost.adml`` file is in the ``C:\Windows\PolicyDefinitions\en-US`` folder.

   .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00008.png

>       :alt: Verify the mattermost.adml file is in the C:\Windows\PolicyDefinitions\en-US folder.

Reviewer: ```suggestion
      :alt: Verify the mattermost.adml file is present in the Windows\PolicyDefinitions\en-US directory.
```

1. Run the ``Edit group policy`` application by clicking ``Start``, typing ``gpedit`` into the search field and clicking on the resulting ``Edit group policy`` search option.

   .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00009.png

>       :alt: Run the Edit group policy application by clicking Start, typing gpedit into the search field and clicking on the resulting Edit group policy search option.

Reviewer: ```suggestion
      :alt: When configuring Mattermost using group policy settings, run the Edit group policy application by going to Start, typing gpedit into the search field, then selecting the resulting Edit group policy search option.
```

3. In the resulting window for ``DefaultServerList``, click on ``Enabled`` to turn the feature on and then click on the ``Show…`` button in the ``Options:`` section of the window to add default servers.

   .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00011.png
   
      :alt: In the resulting window for ``DefaultServerList``, click on ``Enabled`` to turn the feature on and then click on the ``Show…`` button in the ``Options:`` section of the window to add default servers.

4. In the resulting window, add desired Mattermost servers using a memorable name (i.e. Community) and the web url of the Mattermost server (i.e. https://community.mattermost.com).

5. Click the ``OK`` button twice and close the ``Edit group policy`` application.

   .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00012.png

>       :alt: Click the ``OK`` button twice and close the ``Edit group policy`` application.

Reviewer: ```suggestion
      :alt: Select OK twice, then close the Edit group policy application.
```
